lib/foreman_maintain/concerns/logger.rb

```diff
@@ -1,13 +0,0 @@
-require 'logger'
-
-module ForemanMaintain
-  module Concerns
-    module Logger
-      def logger
-        @logger ||= ::Logger.new($stderr).tap do |logger|
```

Comment:
@iNecas ,
Thank you for your views. I will then go ahead with foreman_maintain/concerns/logger.rb and update the pull-request soon.